pkg/osquery/tables/wlan/getbssidps.go

```diff
@@ -0,0 +1,2428 @@
+package wlan
+
+// TODO: move this into a ps1 file for better reading/editing, and load from there.
+const getBSSIDCommandTemplate = `
```

Comment:
Please move this into something with `go-bindata` there are several examples in the packaging code